templates/drupal8/files/README.md

```diff
@@ -1,43 +1,26 @@
-# Drupal project template for Platform.sh
+# Drupal 8 for Platform.sh
 
-This project provides a starter kit for Drupal 8 projects hosted on [Platform.sh](http://platform.sh). It
-is very closely based on the [Drupal Composer project](https://github.com/drupal-composer/drupal-project).
+This template builds Drupal 8 using the [Drupal Composer project](https://github.com/drupal-composer/drupal-project) for better flexibility.  It also includes configuration to use Redis for caching, although that must be enabled post-install in `.platform.app.yaml`.
 
-## Starting a new project
+## Services
 
-To start a new Drupal 8 project on Platform.sh, you have 2 options:
+* PHP 7.2
+* MariaDB 10.2
+* Redis 5
 
-1. Create a new project through the Platform.sh user interface and select "start
-   new project from a template".  Then select Drupal 8 as the template. That will
-   create a new project using this repository as a starting point.
+## Post-install
```

Comment:
This section could vary widely between templates, from nothing at all to including default passwords, etc.